modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1172,8 +1172,9 @@ definition {
 
 	macro goToRelationshipsDetails {
 		Click(
-			key_name = ${fieldName},
-			locator1 = "CreateObject#SELECT_OBJECT_FIELD_IN_TABLE");
+			key_labelName = ${labelName},
+			key_relationHierarchy = ${relationHierarchy},
+			locator1 = "ObjectRelationships#OPTIONS_TAB_RELATIONSHIPS_HIERARCHY_AND_LABEL");
```

Comment:
This change looks risky, how is this going to affect our existing tests that rely on ObjectAdmin.goToRelationshipsDetails?

Reply:
It won't impact. I tested all of them manually after the change.

Reply:
Can you analyze the results of the test suite and check the results of the tests that use this macro? Please make sure they're passing.